tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "target": "es6",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Strictly speaking, JSON have no comments. Is there a `.js` config format for that?

Reply:
https://github.com/microsoft/TypeScript/pull/5450

They could have fixed this lots of different ways (like saying "this is [jsonc](https://komkom.github.io/)".  The fact that comments are accepted isn't documented terribly well in the TSConfig Reference, at least that I could find.

Reply:
> Strictly speaking, JSON have no comments. Is there a .js config format for that?

This is standard output from the tool.  I also find it surprising.

You can see one of these by locally issuing `npm install typescript && npx tsc --init`